ert3/evaluator/_evaluator.py

```diff
@@ -1,25 +1,98 @@
 import asyncio
-import os
 import pathlib
 import shutil
 from collections import defaultdict
 from pathlib import Path
-from typing import Dict, Any, Tuple, List
+from typing import Coroutine, Dict, Any, Tuple, List
 
 
 import cloudpickle
-from pydantic import FilePath
 
 import ert3
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 from ert_shared.ensemble_evaluator.prefect_ensemble import PrefectEnsemble
-
-from ert3.config import EnsembleConfig, StagesConfig, Step
-from ert3.data import EnsembleRecord, MultiEnsembleRecord, RecordTransmitter, Record
+from ert3.config import (
+    EnsembleConfig,
+    StagesConfig,
+    TransportableCommand,
+    Unix,
+    Function,
+    Step,
+)
+from ert3.data import (
+    EnsembleRecord,
+    MultiEnsembleRecord,
+    RealisationsToRecordToTransmitter,
+    RecordTransmitter,
+    Record,
+    SharedDiskRecordTransmitter,
+)
 
 _EVTYPE_SNAPSHOT_STOPPED = "Stopped"
 _EVTYPE_SNAPSHOT_FAILED = "Failed"
+_MY_STORAGE_DIRECTORY = ".my_storage"
+
+CoroutineTransmitters = List[Coroutine[Any, Any, None]]
+
+
+def _add_storage(path: Path) -> Path:
+    if path.name != _MY_STORAGE_DIRECTORY:
+        return path / _MY_STORAGE_DIRECTORY
+    return path
+
+
+def _create_command_transmitter(
+    command: TransportableCommand,
+    path: Path,
+) -> RecordTransmitter:
+    transmitter = SharedDiskRecordTransmitter(command.name, _add_storage(path))
+    with open(command.path, "rb") as f:
+        asyncio.get_event_loop().run_until_complete(
+            transmitter.transmit_data([f.read()])
```

Comment:
I can't find my comment so I am not sure if you responded, but why not use the `gather` at line 73 in this diff for these coros?